huxley/invoice_automation/src/authentication/authenticator.py

```diff
@@ -0,0 +1,67 @@
+import threading
+from concurrent.futures import ThreadPoolExecutor
+from http.server import BaseHTTPRequestHandler, HTTPServer
+from urllib.parse import parse_qs, urlparse
+
+from intuitlib.client import AuthClient
+from intuitlib.enums import Scopes
+
+
+class Authenticator:
+    # valid options: sandbox, production
+    ENVIRONMENT = "production"
+    REDIRECT_URI = "http://localhost:8000"
+    HOST_NAME = "localhost"
+    SOCKET_NUMBER = 8000
```

Comment:
Does this socket number setting work regardless of what server we are deploying to? Wondering if this is a local test-specific decision or if this works fine for production as well.

Reply:
This module was implemented when I thought we were going to do the OAuth2 flow on the huxley server rather than copying the tokens during deployment. However, the socket number was chosen when I was doing local testing, but I don't expect this module to be used unless we change the current deployment flow.

Reply:
Okay that sounds good! Can we remove this part for now then?

Reply:
Yeah I'll remove the authentication module from the commit too